demisto_sdk/commands/generate_modeling_rules/generate_modeling_rules.py

```diff
@@ -0,0 +1,439 @@
+import json
+import logging
+import traceback
+from io import StringIO
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+import typer
+
+from demisto_sdk.commands.common.constants import (
+    FILETYPE_TO_DEFAULT_FROMVERSION,
+    GENERAL_DEFAULT_FROMVERSION,
+    FileType,
+)
+from demisto_sdk.commands.common.logger import setup_rich_logging
+from demisto_sdk.commands.common.tools import get_max_version
+
+app = typer.Typer()
+logger = logging.getLogger("demisto-sdk")
+
+
+@app.command(no_args_is_help=True)
+def generate_modeling_rules(
+    mapping: Path = typer.Argument(
+        ...,
+        exists=True,
+        dir_okay=True,
+        resolve_path=True,
+        show_default=False,
+        help=(
+            "The Path to a csv or tsv file containing the mapping for the modeling rules."
+        ),
+    ),
+    raw_event_path: Path = typer.Argument(
+        ...,
+        exists=True,
+        dir_okay=True,
```

Comment:
the `help` says it's a file, why `dir_okay=True`?

Reply:
Same